crates/bevy_render/src/renderer/render_device.rs

```diff
@@ -45,7 +45,15 @@ impl RenderDevice {
 
     /// Creates a [`ShaderModule`](wgpu::ShaderModule) from either SPIR-V or WGSL source code.
     #[inline]
-    pub fn create_shader_module(&self, desc: wgpu::ShaderModuleDescriptor) -> wgpu::ShaderModule {
+    /// # Safety
+    ///
+    /// we are interfacing with shader code, which may contain undefined behavior,
```

Comment:
These safety comments should be copied from the underlying unsafe wgpu method that this calls :)